hydra-works.gemspec

```diff
@@ -21,6 +21,8 @@ Gem::Specification.new do |spec|
   spec.add_dependency 'hydra-pcdm', '>= 0.8'
   spec.add_dependency 'hydra-derivatives', '~> 3.0'
   spec.add_dependency 'hydra-file_characterization', '~> 0.3', '>= 0.3.3'
+  spec.add_dependency 'active-fedora', '~> 11.0.0.rc7'
```

Comment:
Does this PR **require** AF 11, or would it work without this constraint? I'm concerned about forcing AF 11 on folks before we have tooling in place for them to deal with unordered values.